hugo/content/faqs/_index.md

```diff
@@ -2,87 +2,88 @@
-answer = "TBD"
+1. Go to the menu bar (top-left corner) and open the form for adding a track by selecting `File > Open Track`.
 
-[[faq.section]]
-question = "How do I cite the data files displayed on the Genome Portal?"
-answer = "TBD"
+2. Use the action button (circular `+`) inside the track selector (bottom-right corner) to access the `Add track` form, where you can provide a URL or select a local file to load. More details on <a href="https://jbrowse.org/jb2/docs/tutorials/config_gui/#adding-a-track" target="_blank">adding a data track</a> can be found in the JBrowse User Guide.
 
-[[faq.section]]
-question = "How do I cite the genome browser used in the Genome Portal?"
-answer = "TBD"
+*Note that for either of these options to work, your data must have the same genomic coordinates as the genome assembly available on the Genome Portal.*
+{{< /faq_block >}}
 
-[[faq.section]]
-question = "How can I indicate in the Data Availability Statement of my manuscript that my data can be visualised on the Genome Portal?"
-answer = "TBD"
+{{< faq_block title="Can unpublished data be displayed on the Genome Portal?" >}}
+No, we require data be publicly available. However, we can begin adding data under embargo that will soon become available once the manuscript is accepted for publication. More details can be found in the next question.
+{{< /faq_block >}}
+
+{{< faq_block title="Is it possible to add my data to the Genome Portal while a manuscript is under review?" >}}
+Yes, we accept data under embargo expected to become available after the manuscript under review is approved for publication. The data should be deposited in a public repository and have a reserved DOI and/or accession number. This allows you to indicate in the Data Availability Statement of your manuscript that your data can be visualised on the Genome Portal. Planning is key! Reach out to us via email <dsn-eb@scilifelab.se> or through the <a href="/contact" target="_blank">Contact</a> form as soon as you wish to start this process.
+{{< /faq_block >}}
+
+{{< faq_block title="How long does it take for my data to be displayed on the Genome Portal?" >}}
+Once we have received the form listing the links and metadata associated with your genomic datasets, the webpage implementation process generally takes around one week. Please note that this may take a bit longer during public holidays and summer.
+{{< /faq_block >}}
+
+#### About the genome browser
+
+{{< faq_block title="Why is JBrowse the genome browser used in the Genome Portal?" >}}
+We chose JBrowse for several reasons. It is a robust genome browser with powerful features for visualizing genomic data. It receives active maintenance and support from both system developers and the research community. Additionally, it is highly customizable and allows for the creation of new view types via a plugin system, making it more than a genome browser—it serves as a versatile platform for development
```

Comment:
I suggest to specify that we are using JBrowse 2; this has come up in the review of the other pages and then we said that it is better to be consistent and always specify the number when we write about the browser. 

Added also that it is open-source, and put a full stop at the end since to harmonize with the formatting of the other paragraphs
```suggestion
We chose JBrowse 2 for several reasons. It is a robust, open-source genome browser with powerful features for visualizing genomic data. It receives active maintenance and support from both system developers and the research community. Additionally, it is highly customizable and allows for the creation of new view types via a plugin system, making it more than a genome browser — it serves as a versatile platform for development.